src/OrbitGl/FrameTrack.cpp

```diff
@@ -199,8 +199,8 @@ std::string FrameTrack::GetTooltip() const {
       GetPrettyTime(absl::Nanoseconds(stats_.average_time_ns())));
 }
 
-std::string FrameTrack::GetBoxTooltip(PickingId id) const {
-  const TextBox* text_box = time_graph_->GetBatcher().GetTextBox(id);
+std::string FrameTrack::GetBoxTooltip(Batcher* batcher, PickingId id) const {
```

Comment:
Why does the `GetBoxTooltip` receive a `Batcher*`, `GetSampleTooltip` a `Batcher&`?  If the parameter is intended to be not nullptr, `Batcher&` is preferrable.

Reply:
Thanks for pointing out. Both should be const &. I tried but used to have a problem with some method that weren't const.

I've modified it now.